src/components/NewLocationPage/CountyMap/CountyMap.tsx

```diff
@@ -61,6 +61,24 @@ const MAP_TYPE_INFO: { [key in MapType]: MapTypeInfo } = {
   },
 };
 
+function useMapTileOffset(): number {
```

Comment:
Consider adding a comment to this function explaining its purpose (to avoid the map tile overlapping the footer), since that's not obvious from the name (and renaming to `useMapTileOffsetAvoidingFooterOverlap()` seems a bit overkill.

Also, do we call it a "map tile" in other parts of the code?  Or should we change this to `MapContainer` or `CountyMap` or similar?

Reply:
I think the word "tile" was just used in the Trello card. I'll rename this to `useMapContainerOffset` and add a comment above the function.